GetBPDTransactions/handler.ts

```diff
@@ -52,7 +52,13 @@ export const toApiBPDTransactionList = (
     transactions: domainObj.map(transaction => {
       return {
         ...transaction,
+        acquirer_descr: transaction.acquirer, // TODO: It's an enum type
+        amount_currency_descr: "EUR", // TODO: fixed to 978 = EUR
+        circuit_type_descr: transaction.circuit_type, // TODO: It's an enum type
         insert_date: transaction.insert_date?.toISOString(),
+        operation_type_descr: transaction.operation_type, // TODO: It's an enum type
```

Comment:
We leave the `_descr` field for future extensions? At this moment are the same as no `_descr` field.

Reply:
Yes `_descr` will be field showed by frontend for human readable values. For example this will be the match for circuit_type <-> circuit_type_descr

Circuit Payment:
circuit_type - circuit_type_descr
00 – Pagobancomat 
01 - Visa
02 - Mastercard
03 - Amex
04 - JCB
05 - UnionPay
06 - Diners
07 - Codice PostePay
08 - BancomatPay
09 - SatisPay
10 - private circuit (onus, owen)
xx - future use

Reply:
Ok, we can extend these field on another PR